src/System.IO.FileSystem.Primitives/src/System.IO.FileSystem.Primitives.csproj

```diff
@@ -4,24 +4,23 @@
   <PropertyGroup>
     <ProjectGuid>{6C05678E-394C-4CFF-B453-A18E28C8F2C3}</ProjectGuid>
     <AssemblyName>System.IO.FileSystem.Primitives</AssemblyName>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
```

Comment:
You need to bump the assembly version and the target framework. for this library. See my docs which are still in PR https://github.com/dotnet/corefx/pull/12036, and please add feedback to that PR if you have further questions.